Overview.bs

```diff
@@ -1014,6 +1057,73 @@ If the font load or extension has failed the client should choose one of the fol
 Regardless of which of the above options are used, the saved client state for this font must be
 discarded.
 
+### Load a Font in a User Agent with a HTTP Cache ### {#load-a-font}
+
+<em>This section is not normative.</em>
+
+The previous section [[#extend-subset]] provides no guidance on how a user agent should handle
```

Comment:
Non-normative MAY seems a bit weak here. 

I would prefer 

> User agents which implement [[fetch]] should use this algorithm to save client
> state to the user agents HTTP cache ([[RFC7234]]).

and drop the non-normative.